pkg/ddl/table.go

```diff
@@ -239,8 +239,8 @@ func onCreateTables(d *ddlCtx, t *meta.Meta, job *model.Job) (int64, error) {
 	// We don't construct jobs for every table, but only tableInfo
 	// The following loop creates a stub job for every table
 	//
-	// &*job clones a stub job from the ActionCreateTables job
-	stubJob := &*job
+	// it clones a stub job from the ActionCreateTables job
+	stubJob := job.Clone()
```

Comment:
does `CtxVars` matters here? it's not cloned

Reply:
`CtxVars` is only used by "reorg partition" and "rename tables", it is not used in "create tables".